src/main/java/org/terasology/surfacefacets/examples/SurfaceFacetsExampleRasterizer.java

```diff
@@ -69,9 +69,9 @@ public void generateChunk(CoreChunk chunk, Region chunkRegion) {
         SurfaceNormalFacet surfaceNormalFacet = chunkRegion.getFacet(SurfaceNormalFacet.class);
         SurfaceSteepnessFacet surfaceSteepnessFacet = chunkRegion.getFacet(SurfaceSteepnessFacet.class);
 
-        for (Vector3i position : chunkRegion.getRegion()) {
-            Vector3i blockPosition = ChunkMath.calcRelativeBlockPos(position);
-            if (surfacesFacet.getWorld(JomlUtil.from(position))) {
+        for (Vector3ic position : chunkRegion.getRegion()) {
+            Vector3i blockPosition = ChunkMath.calcRelativeBlockPos(position, new Vector3i());
```

Comment:
I don't think we need this here, as we're not storing or modifying the value anywhere.